example_1/bringup/launch/rrbot.launch.py

```diff
@@ -14,15 +14,24 @@
 
 
 from launch import LaunchDescription
-from launch.actions import RegisterEventHandler
+from launch.actions import DeclareLaunchArgument, RegisterEventHandler
+from launch.conditions import IfCondition
 from launch.event_handlers import OnProcessExit
-from launch.substitutions import Command, FindExecutable, PathJoinSubstitution
+from launch.substitutions import Command, FindExecutable, LaunchConfiguration, PathJoinSubstitution
 
 from launch_ros.actions import Node
 from launch_ros.substitutions import FindPackageShare
 
 
 def generate_launch_description():
+    start_rviz = LaunchConfiguration("start_rviz")
+
+    start_rviz_arg = DeclareLaunchArgument(
```

Comment:
We usually use different syntay to declare arguments. And also declaring them at the top of the function. Plase check other files and adjust style approprately.